extensions/ql-vscode/test/vscode-tests/cli-integration/legacy-query.test.ts

```diff
@@ -111,11 +111,17 @@ describeWithCodeQL()("using the legacy query server", () => {
   let qs: qsClient.QueryServerClient;
   let cliServer: cli.CodeQLCliServer;
 
+  let supportNewQueryServer = true;
+
   beforeAll(async () => {
     const extension = await getActivatedExtension();
     cliServer = extension.cliServer;
     cliServer.quiet = true;
 
+    if (!(await cliServer.cliConstraints.supportsNewQueryServerForTests())) {
+      supportNewQueryServer = false;
```

Comment:
Could you add a logging statement here saying something about how even though the tests have passed, they haven't run anything because the CLI doesn't support the new query server.

Reply:
Thanks for the suggestion!  Done.